src/core/Akka.MultiNodeTestRunner/Discovery.cs

```diff
@@ -106,20 +98,11 @@ internal static ConstructorInfo FindConfigConstructor(Type configUser)
             var current = configUser;
             while (current != null)
             {
-
-#if CORECLR
-                var ctorWithConfig = current
```

Comment:
I think this is probably fine, but I'll need to see the feedback from the MNTR for certain